datasets/mx/governors/crawler.py

```diff
@@ -0,0 +1,124 @@
+import re
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+def extract_birth_place_and_date(link_governor_page, context: Context):
+    """
+    The birth place and date can be found in the page of each governor
+    in a ul tag. It is not always in the same position in the list,
+    so we will iterate over all elements and find the one that has
+    the string "Lugar y fecha de nacimiento:". If it is not found,
+    we raise a warning and return None.
+
+    :param link_governor_page: Link to a page of a specific governor
+    :param context: The context object.
+    """
+
+    response = context.fetch_html(link_governor_page)
+
+    path_to_information = '//*[@id="fContenido"]/div/div/div[2]/div/div/div/div[2]/div/div[1]/ul/li'
+
+    for element in response.xpath(path_to_information):
+        text = ' '.join(element.itertext())
+        if 'Lugar y fecha de nacimiento:' in text:
+            pattern = r'Lugar y fecha de nacimiento: ([^.]+\.) (\d{1,2}/\d{1,2}/\d{4})\.'
+            match = re.search(pattern, text)
+            if match:
+                birth_place = match.group(1)
+                birth_date = match.group(2)
+                return birth_place, birth_date
+
+    context.log.warning("Failed to identify birth place and date, link: {}".format(link_governor_page))
+    return None, None
+
+
+def crawl_item(input_html, link_governor_page, context: Context):
+    """
+    Creates an entity, a position and a occupancy from the raw HTMLElement.
+
+    :param input_html: The div representing that governor
+    :param link_governor_page: Link to that governor page
+    :param context: The context object.
+    """
+
+    # The easiest way to get the data from the HTML is surprisingly from the text of the class
+    # All the information in the web site is written there in the form:
+    # col-xs-12 col-sm-6 col-md-4 pb15 mix {name of the governor} [{state}] {start_date} {end_date}
+    # We will then use a regex to extract the name, state, start_date and end_date
+
+    regex_pattern = r'col-xs-12 col-sm-6 col-md-4 pb15 mix (.*?) \[(.*?)\] (\d{2}/\d{2}/\d{4}) (\d{2}/\d{2}/\d{4})'
```

Comment:
Puttng all the data including dates formatted like this in the class is just wild. But let me not judge them. Does it all work if we do this, to tighten up the regex a little more and be sure we'll find out if they change things around?
```suggestion
    regex_pattern = r'^col-xs-12 col-sm-6 col-md-4 pb15 mix (.*?) \[(.*?)\] (\d{2}/\d{2}/\d{4}) (\d{2}/\d{2}/\d{4})$'
```